lib/components/molecules/ArticleThumbnail/ArticleThumbnail.js

```diff
@@ -21,7 +26,7 @@ const ArticleThumbnail = ({
   anchorProps,
   ...otherProps
 }: Props): Node => {
-  let thumbImage = <Image {...image} />;
+  let thumbImage = <Image key={`${image.alt}_${i}`} {...image} />;
```

Comment:
This is wrong, there is no need to use a key here
The key should unique and contextual

Reply:
all changes have been roll backed